test/e2e/storage/csi_mock_volume.go

```diff
@@ -772,17 +772,23 @@ var _ = utils.SIGDescribe("CSI mock volume", func() {
 
 		createVolume := "CreateVolume"
 		deleteVolume := "DeleteVolume"
-		publishVolume := "NodePublishVolume"
+		// publishVolume := "NodePublishVolume"
 		unpublishVolume := "NodeUnpublishVolume"
-		stageVolume := "NodeStageVolume"
+		// stageVolume := "NodeStageVolume"
 		unstageVolume := "NodeUnstageVolume"
 
 		// These calls are assumed to occur in this order for
-		// each test run.
+		// each test run. NodeStageVolume and
+		// NodePublishVolume should also be deterministic and
+		// only get called once, but sometimes kubelet calls
+		// both multiple times, which breaks this test
+		// (https://github.com/kubernetes/kubernetes/issues/90250).
+		// Therefore they are temporarily commented out until
+		// that issue is resolved.
 		deterministicCalls := []string{
 			createVolume,
-			stageVolume,
```

Comment:
Do we need the test to have strict ordering checks? I believe other test cases in this file only check for >= count.

Reply:
All of these tests also check for strict ordering of calls: https://github.com/kubernetes/kubernetes/blob/5aa3805a5ff11860437510ff9b4115c7a0afae9d/test/e2e/storage/csi_mock_volume.go#L592

That's how @jsafrane wrote the `compareCSICalls`. I find this useful.